packages/wdio-cli/tests/commands/config.test.ts

```diff
@@ -104,28 +104,44 @@ describe('install compliant NPM tag packages', () => {
     test('it should install tagged version if cli is tagged with beta', async () => {
         setFetchSpec('beta');
         (inquirer.prompt as any as jest.Mock).mockReturnValue(Promise.resolve(args))
+        // @ts-expect-error
+        fs.promises = { writeFile: jest.fn()
```

Comment:
is these are all the same then you can define it once on top or place it in the beforeEach instead unless the other tests would be affected by it

Reply:
Agree, let's move this up to the top of the file and call `fs.promises.writeFile.mockClear()` in `beforeEach`.

Reply:
WIll look into this